configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,28 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    service_hash = [{
+      :directory_ref => "bindir",
+      :id => "MCOService",
+      :guid => "7601FCEA-90B3-CC69-6A69-4087FBC7292D",
+      :win64 => true,
+      :file => {
+        :id => "RubyWExe",
+        :source => 'stagedir\\sys\\ruby\\bin\\rubyw.exe',
+      },
+      :serviceinstall => {
+        :id => "MCOServiceInstaller",
+        :description => "Puppet Labs server orchestration framework",
+        :displayname => "Marionette Collective Server",
+        :name => "mcollective",
+        :start => "demand",
+        :arguments => '-I&quot;[INSTALLDIR]puppet\\lib;[INSTALLDIR]facter\\lib;[INSTALLDIR]hiera\\lib;[INSTALLDIR]mcollective\\lib&quot; -rubygems &quot;[INSTALLDIR]mcollective\\bin\\mcollectived&quot; --daemonize',
```

Comment:
This also needs to be made into a variable, but I haven't decided what that's going to look like yet